src/browser-attributes-resource.ts

```diff
@@ -10,13 +11,19 @@ export const computeScreenSize = (screenWidth: number): ScreenSize => {
   return 'unknown';
 };
 
+export const computeBrowserName = (userAgent: string) => {
+  const uaParser = new UAParser(userAgent);
+  return uaParser.getBrowser().name ?? 'unknown';
+};
+
 export function configureBrowserAttributesResource(): Resource {
   return new Resource({
     'user_agent.original': navigator.userAgent,
     //https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent#mobile_tablet_or_desktop
     'browser.mobile': navigator.userAgent.includes('Mobi'),
     'browser.touch_screen_enabled': navigator.maxTouchPoints > 0,
     'browser.language': navigator.language,
+    'browser.name': computeBrowserName(navigator.userAgent),
```

Comment:
Wondering if we should also include the browser version since we have access to it?

Reply:
sure, yeah, I can do that